### PR TITLE
Fix: Improve webpage responsiveness to scrolling

### PR DESCRIPTION
**Pull request by RajeshTechForge**
## Related Issue
**Issue**: It takes a while for the webpage to react to my scrolling #183

## Description
The website exhibits slow or unresponsive scrolling behaviour. There is a noticeable delay between the user's scrolling action and the webpage content updating. This can be described as lag, jerkiness, or a general feeling of sluggishness.

I’ve identified that the pull request on issue #109 is the root cause of this bug. I have removed the problematic section of the code.

## Type of PR

- [x] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
None

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [ ] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
None

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -68,15 +68,6 @@ const App = () => {
...
 
-  useEffect(() => {
-    const lenis = new Lenis();
-    function raf(time) {
-      lenis.raf(time);
-      requestAnimationFrame(raf);
-    }
-    requestAnimationFrame(raf);
-  }, []);
-
```
Comment (Harshil-Jani) [inline src/main.jsx:79]: This just removes the use of Lenis completely right ?

Comment (RajeshTechForge) [reply src/main.jsx:79]: Yes

Comment (Harshil-Jani) [reply src/main.jsx:79]: Then let's do one thing. Remove it completely as dependency. Let's keep it very very simple without lenis at all. Remove it from package.json and from imports too.

Comment (RajeshTechForge) [reply src/main.jsx:79]: I am on the way to commit Changes.

**Pull request closed without merging**